# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/credentialmanagement/identity/tests/browser/browser_single_concurrent_identity_request.js
# Commit: d07def1365a4
# Full Hash: d07def1365a49dc8ecbf5b6c10fdfbd5ec750932
# Author: pstanciu <pstanciu@mozilla.com>
# Date: 2025-06-11 18:38:51
# Regressor Bug: 1969017
# File Overlap Count: 2
# Description:
#   Revert "Bug 1969015, 1969017: apply code formatting via Lando" for causing WPT failures @lfedcm-identity.create-store-collect.tentative.sub.https.html
#   
#   This reverts commit 03d4474e360f28be0fb075379f4e4535bd34b252.
#   
#   Revert "Bug 1969017 - Refactor FedCM to use a managed actor rather than tagging onto WindowGlobalParent - r=anti-tracking-reviewers,emz"
# ==============================================================================

diff -r 4e25ddaadc86 -r d07def1365a4 dom/credentialmanagement/identity/tests/browser/browser_single_concurrent_identity_request.js
--- a/dom/credentialmanagement/identity/tests/browser/browser_single_concurrent_identity_request.js	Wed Jun 11 13:42:13 2025 +0000
+++ b/dom/credentialmanagement/identity/tests/browser/browser_single_concurrent_identity_request.js	Wed Jun 11 18:04:27 2025 +0300
@@ -44,8 +44,8 @@
   ok(concurrentResponse.name, "expect a DOMException which must have a name.");
   is(
     concurrentResponse.name,
-    "NotAllowedError",
-    "Expected 'NotAllowedError', but got '" + concurrentResponse.name + "'"
+    "InvalidStateError",
+    "Expected 'InvalidStateError', but got '" + concurrentResponse.name + "'"
   );
 
   // Close tabs.